### PR TITLE
feat(metadata): add db for hostname and tags

### PR DESCRIPTION
**Pull request by rsdmike**

### CODE REVIEW

File: src/models/Config.d.ts [hunk 1]
```diff
@@ -39,6 +40,7 @@ export interface configType {
   cors_origin: string
   cors_headers: string
   cors_methods: string
+  connection_string: string
```
Comment (brianosburn-intel) [inline src/models/Config.d.ts:43]: should this be optional?

Comment (rsdmike) [reply src/models/Config.d.ts:43]: hmm, maybe -- i supposed we can eliminate the "use_db" option and just check if connection string is null or empty? That might be better option? what do ya think?

**Review by brianosburn-intel — APPROVED** [review]
looks good

File: src/models/Config.d.ts (hunk 1 above)
Comment (brianosburn-intel) [reply src/models/Config.d.ts:43]: personally, i like the separate bool to disable the db. then we could have a default localhost string in the config. might be a little less intuitive if we go with just the connection string.

**Comment by rsdmike** [comment]
Address #53

**Pull request merged**